### PR TITLE
ref: Don't use eventstore for deletions

### PR DESCRIPTION
**Pull request by lynnagara**
Make a raw query instead of using eventstore for event data deletions.
Since we only need 2 columns here and not the entire event, we want to
skip the unnecessary nodestore fetch that will soon be performed by
eventstore.

### CODE REVIEW

**Comment by fpacifici** [comment]
I am a bit concerned with this way. This PR is saying that we should use either snuba or eventstore to load events. Which is the behavior we wanted to remove by adding eventstore.
1) is there a specific reason for using snuba in this case (like production performance problem) or is it speculation ?
2) do you know about other use cases that would turn to this approach ? That would be bad
3) could we revisit the topic whether eventstore should offer only one event interface or whether it should allow the client to specify which fields to load ? If the whole event is needed in 99% of the case we may be able to accomodate this case still through eventstore.

File: src/sentry/deletions/defaults/group.py [hunk 1]
```diff
@@ -25,34 +26,33 @@ def chunk(self):
...
                     ],
                 ]
             )
 
-        events = eventstore.get_events(
-            filter=eventstore.Filter(
-                conditions=conditions, project_ids=[self.project_id], group_ids=[self.group_id]
-            ),
-            limit=self.DEFAULT_CHUNK_SIZE,
+        events = raw_query(
```
Comment (markstory) [inline src/sentry/deletions/defaults/group.py]: An alternative approach to using `raw_query` could be to use `discover.query`. It is a high-level API that will let you select the subset of columns that you want.

**Comment by lynnagara** [comment]
The purpose of this PR is to skip an unnecessary nodestore load for each page of events (10,000 per page) . Since we only want a list of event IDs here any not the actual Event object, it seems ok that we don't go through eventstore. It's possible that loading the events from nodestore has no noticable impact but I was trying to be safe here. Re 2: There aren't any other use cases right now that are like this.

**Comment by fpacifici** [comment]
About deletions. We are already sending a delete_multi to nodestore of all the events we would have loaded right before, so I doubt this is a deal breaker (https://github.com/getsentry/sentry/blob/master/src/sentry/deletions/defaults/group.py#L52)

I think the problem is at a higher level. eventstore provides an interface to load a list of events given some conditions, these can easily return a vast number of events.  I think we should still have a way to return a simplified view of the event and allow the user to bind the rest afterwards.

**Pull request closed without merging**